### PR TITLE
Add link to upgrade screen to settings

### PR DESCRIPTION
**Pull request by johnschoeman**
Why:
We would like for users to be able to access the upgrade screen without
needing to create and edit a habit first.

This commit:
Adds a button to the settings screen that will show the upgrade screen
modal. We also updated the copy and design of the upgrade screen to be
more agnostic to how a user arrived at the screen.

|Before|After|
|---|---|
|Simulator Screen Shot - iPhone 12 Pro Max - 2021-02-25 at 07 05 39|Simulator Screen Shot - iPhone 12 Pro Max - 2021-02-25 at 06 47 39|

### CODE REVIEW

File: src/Settings.tsx [hunk 1]
```diff
@@ -10,92 +10,115 @@ import {
...
+              title="Terms of Use"
+              onPress={handleOnPressTermsOfUse}
+            />
+            {sessionStore.subscription === "Free" && (
+              <ListItemButton
+                title="Upgrade to Plus"
+                onPress={handleOnPressUpgrade}
+              />
+            )}
+            {sessionStore.subscription === "Free" && (
```
Comment (devinjameson) [inline src/Settings.tsx:99]: Nice catch on these!

File: src/Upgrade.tsx [hunk 2]
```diff
@@ -173,15 +214,10 @@ const style = StyleSheet.create({
...
-  contentContainer: {
-    flexGrow: 1,
-    paddingTop: Sizing.layout.x20,
-    paddingBottom: Sizing.layout.x60,
-    paddingHorizontal: Sizing.layout.x20,
-  },
   innerContainer: {
-    flexGrow: 1,
     justifyContent: "space-between",
+    paddingTop: Sizing.layout.x20,
```
Comment (devinjameson) [inline src/Upgrade.tsx:219]:
```suggestion
    paddingTop: Sizing.x20,
```
And for any other cases where we use `layout`.

File: src/Upgrade.tsx [hunk 3]
```diff
@@ -199,68 +235,54 @@ const style = StyleSheet.create({
...
-    ...Buttons.bar.primary,
-    paddingVertical: Sizing.layout.x20,
-    width: "100%",
+  policyLinksContainer: {
+    flexDirection: "row",
+    justifyContent: "center",
   },
-  buttonText: {
-    ...Buttons.barText.primary,
+  policyLinkContainer: {},
```
Comment (devinjameson) [inline src/Upgrade.tsx:279]: Needed?

**Review by devinjameson — APPROVED** [review]
Code looks great! Few thoughts:
- I notice we use `alignSelf: "center"` in a few places. Would it make sense to wrap all these nodes in a `View` that centers them all, if they're colocated?
- Thoughts on making the 'Plus Package' text a little smaller? It feels a bit too big to my eye.
- Thoughts on bolding `Building Habits` to provide more contrast to the subhead?
- Thoughts on centering everything vertically?
- Thoughts on using `month` vs. `Month`?

Hope that's helpful!

File: src/Upgrade.tsx (hunk 2 above)
Comment (johnschoeman) [reply src/Upgrade.tsx:219]: dont currently have this in our styles. was planning on taking a whole pass at all of em in a separate pr.

**Pull request merged**